src/internal/utils/bucketHelper.ts

```diff
@@ -399,6 +399,18 @@ export function getMeasuresFromMdObject(mdObject: VisualizationObject.IVisualiza
     }, []);
 }
 
+export function getBucketItemsByTypeFromMdObject(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
+    type: string,
+): VisualizationObject.BucketItem[] {
+    return get(mdObject, "buckets", [])
```

Comment:
what about just getting buckets from mdObj and then using https://github.com/gooddata/gooddata-react-components/blob/master/src/helpers/mdObjBucketHelper.ts#L13

Reply:
Fixed